test/check-application

```diff
@@ -814,6 +814,9 @@ class TestFiles(testlib.MachineCase):
         b.mouse("[data-item='delete2']", "click", ctrlKey=True)
         b.wait_visible("[data-item='delete1'].row-selected")
         b.wait_visible("[data-item='delete2'].row-selected")
+        b.focus("#files-card-parent")
+        # For strange reasons ctrlKey remains pressed after the b.mouse() above (spotted in firefox)
```

Comment:
So we keep seeing this in the keyboard event, sounds like something we should investigate in Bidi